### PR TITLE
Update from voxpupuli modulesync_config; Drop RHEL 6; Drop Puppet 5

### PR DESCRIPTION
**Pull request by TheMeier**
#### Pull Request (PR) description
modulesync

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,26 +9,14 @@ on:
...
-          # Configure secrets here:
-          #  https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets
-          BLACKSMITH_FORGE_USERNAME: '${{ secrets.PUPPET_FORGE_USERNAME }}'
-          BLACKSMITH_FORGE_API_KEY: '${{ secrets.PUPPET_FORGE_API_KEY }}'
-        run: bundle exec rake module:push
+  release:
+    name: Release
+    uses: voxpupuli/gha-puppet/.github/workflows/release.yml@v1
+    with:
+      allowed_owner: 'TheMeier'
```
Comment (bastelfreak) [inline .github/workflows/release.yml:17]: that should be `voxpupuli` :)

Comment (bastelfreak) [inline .github/workflows/release.yml]:
```suggestion
      allowed_owner: 'voxpupuli'
```

Comment (TheMeier) [reply .github/workflows/release.yml]: oops fixed

**Comment by bastelfreak** [comment]
thanks for the work. can you please rebase so the merge commit disappears?

**Pull request merged**